src/main/java/uk/gov/ons/census/action/model/entity/Case.java

```diff
@@ -1,12 +1,14 @@
 package uk.gov.ons.census.action.model.entity;
 
 import com.vladmihalcea.hibernate.type.json.JsonBinaryType;
+import java.time.OffsetDateTime;
 import java.util.UUID;
 import javax.persistence.*;
+import javax.persistence.Entity;
```

Comment:
line above should have been removed by the formatter - weird that it hasn't been